src/components/appcontent.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC, useState } from 'react';
+import DashboardIcon from '../assets/dashboard.svg';
+import DataUsageIcon from '../assets/data_usage.svg';
+import DCOpenOrdersIcon from '../assets/dc_open_orders.svg';
+import LeaderboardIcon from '../assets/leaderboard.svg';
+import LegendToggleIcon from '../assets/legend_toggle.svg';
+import Navbar from './common/navbar';
+import SearchField from './common/searchfield';
+import SidePaneGrid from './common/sidepanegrid';
+import SidePaneList from './common/sidepanelist';
+
+const MENU_LIST = [
+    { id: 1, name: "Home", icon: DashboardIcon },
+    { id: 2, name: "OPM", icon: LeaderboardIcon },
+    { id: 3, name: "OPM Comparison", icon: DataUsageIcon },
+    { id: 4, name: "Session", icon: LegendToggleIcon },
+    { id: 5, name: "DC Open Orders", icon: DCOpenOrdersIcon }
+]
+
+type Props = {
+    showSidePane: boolean;
+    showNavbar: boolean;
+    appContent: any;
+};
+
+const AppContent: FC<Props> = ({ showSidePane, showNavbar, appContent }) => {
+    const [showSidePaneGrid, setShowSidePaneGrid] = useState<boolean>(false);
+    const [openSearchField, setOpenSearchField] = useState<boolean>(false);
```

Comment:
Why not move lines 27 and 28 into NavBar component itself.

Reply:
These states are used to apply some styles and display search component in appcontent.tsx